packages/design-system/src/components/popover.tsx

```diff
@@ -22,7 +22,7 @@ const StyledContent = styled(PopoverPrimitive.Content, panelStyles, {
   backgroundColor: "white",
   minWidth: 200,
   minHeight: theme.spacing[13],
-  maxWidth: 265,
+  maxWidth: "max-content",
```

Comment:
What is the idea of `maxWidth: "max-content"`? I see it in many places. What does this do, what the intent as developer?

Reply:
the container doesn't grow beyond the needs of its content, e.g. a block container would grow otherwise full available width

Reply:
So in this case content of the tooltip determines it's width